lib/user-interface/react/src/components/chatbot/DocumentSummarizationModal.tsx

```diff
@@ -230,9 +232,9 @@ export function DocumentSummarizationModal ({
                                 const model = allModels.find((model) => model.modelId === value);
                                 if (model) {
                                     if (!model.streaming && chatConfiguration.sessionConfiguration.streaming) {
-                                        setChatConfiguration({...chatConfiguration, sessionConfiguration: {...chatConfiguration.sessionConfiguration, streaming: false }});
+                                        setChatConfiguration({ ...chatConfiguration, sessionConfiguration: { ...chatConfiguration.sessionConfiguration, streaming: false } });
                                     } else if (model.streaming && !chatConfiguration.sessionConfiguration.streaming) {
-                                        setChatConfiguration({...chatConfiguration, sessionConfiguration: {...chatConfiguration.sessionConfiguration, streaming: true }});
+                                        setChatConfiguration({ ...chatConfiguration, sessionConfiguration: { ...chatConfiguration.sessionConfiguration, streaming: true } });
                                     }
```

Comment:
I'm not sure I fully understand these lines but I feel like this could be simplified maybe? I know you just updated the formatting but I **think** this is equivalent logic…

```
if (model.streaming != chatConfiguration.sessionConfiguration.streaming) {
   setChatConfiguration({ ...chatConfiguration, sessionConfiguration: { ...chatConfiguration.sessionConfiguration, streaming: model.streaming } });
}
```